subprojects/common/slf4j-logger/src/main/kotlin/com/avito/logger/destination/VerboseDestination.kt

```diff
@@ -0,0 +1,42 @@
+package com.avito.logger.destination
+
+import com.avito.logger.LogLevel
+import com.avito.logger.LoggingDestination
+
+data class VerboseMode(
+    val verbosity: LogLevel,
+    val printStackTrace: Boolean
+)
+
+/**
+ * See Logging.md#Verbose-mode
+ */
+class VerboseDestination(private val verboseMode: VerboseMode) : LoggingDestination {
```

Comment:
Everything is great except the name. The verbose mode has intention only with disabled Kibana and that controversial.
Verbose is actual StdoutDestination with implicit dependency on quiet in Gradle

Reply:
yes, i agree; can you suggest a better naming here?

Reply:
Let's name it with what it actually does.
`PrintlnDestination` it's not perfect but clear